pkg/customresource/actionset.yaml

```
@@ -325,6 +341,22 @@ spec:
           type: string
           description: Progress of completion in percentage
           jsonPath: .status.progress.percentCompleted
+        - name: SizeDownloadedB
+          type: integer
+          description: Amount of data downloaded during action execution
```

```suggestion
          description: Amount of data that has been downloaded
```
what do you think about this?

---

if you agree, we can change other columns as well.